docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '2'
+services:
+  sdw_depositor:
+    build:
+      context: .
+      dockerfile: Dockerfile.cdot
+    environment:
+      DOCKER_HOST_IP: ${DOCKER_HOST_IP}
+      SDW_EMAIL_LIST: ${SDW_EMAIL_LIST}
+      SDW_EMAIL_FROM: ${SDW_EMAIL_FROM}
+      SDW_API_KEY: ${SDW_API_KEY}
+      SDW_DESTINATION_URL: ${SDW_DESTINATION_URL}
+      SPRING_MAIL_HOST: ${SPRING_MAIL_HOST}
+      SPRING_MAIL_PORT: ${SPRING_MAIL_PORT}
+      KAFKA_TYPE: ${KAFKA_TYPE}
+      CONFLUENT_KEY: ${CONFLUENT_KEY}
+      CONFLUENT_SECRET: ${CONFLUENT_SECRET}
+    depends_on:
+      - ode
```

Comment:
There is no corresponding ode image in this file to declare the "depends_on" value.

Reply:
This dependency declaration has been removed.